---
layout: post
title: "Pre-print Review of 'Gut microbiota density influences host physiology and is shaped by host and microbial factors'"
author: "PD Schloss"
date: "March 23, 2018"
comments: false
---

The manuscript by [Contijoch and colleagues](https://doi.org/10.1101/277095) presents a very intriguing collection of experiments that evaluate the variation in DNA density within the fecal material of sixteen mammalian species[^1]. I am excited about this work because it highlights that microbial density may be a confounding variable in microbiome studies. Although it would be difficult to ascertain by this method, I have wondered whether how much of a disease like cystic fibrosis is driven by bacterial density rather than a specific set of pathogens. Although I think this is an important contribution to the field, I have several concerns about the methods and the interpretation of the data. Needless to say, the experiments and analysis have really piqued my curiosity.

1. To throw a wet blanket on the analysis, I could argue that the differences and changes in density are of questionable biological significance. If we assume that DNA density is a proxy for density, then the differences the authors see are much less than a single log in density. Although the numbers are a bit questionable, if we assume that typical feces has 10^12 bacteria per gram, a change in 10-50% would still leave a significant amount of microbial biomass. It would be interesting to get the authors' thoughts on their data from a cell count perspective rather than the more abstract DNA density. I am curious what they think is a biologically meaningful difference in density.

2. The authors appear to assume that fecal DNA is coming from living organisms. However, previous studies have indicated that a minority of bacteria in feces are from intact cells. Ben-Amor and colleagues found that 49% of cells were intact, 19% were injured or damaged cells, and 32% were dead (doi: [10.1128/AEM.71.8.4679-4689.2005](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1183343/)). This is important because it would impact their transfer experiments and it may be possible that different mammalian species have different fractions of live/dead bacteria in their feces. In spite of this potential confounder, it is still interesting that the DNA load varies so much across species and between individuals of the same species.

3. The authors did not comment on the fact that some mammalian species vary widely in their DNA density. The density in rats, pigs, and mice varies more than their median density. It would be nice to know whether all individuals within the species consumed the same diet or whether there were other differences that may account for intra-species variation.

4. I fear the authors may have over interpreted the meaning of the variation in carrying capacity. They imply that the carrying capacity is an intrinsic variable for each species. I wonder if it isn't also a product of the taxa within the sample. Some taxa may have "sharp elbows" and exclude other taxa and their density. It would be interesting to see something like a Mantel test relating the difference in density to the difference in beta-diversity within each species. Is density related to community structure?

5. Throughout the manuscript, the authors describe differences in "community fitness". The authors need to provide a better definition of fitness in this context. I am unclear whether it's a measure of the ability of a transferred community to have the same carrying capacity as the host (or the donor) or whether it's a measure of something else. Regardless, I worry that if it is tied to the ability to colonize a germ free animal of a different species that it is a poor metric. Again, we know that much of the DNA in feces is from dead bacteria, there is host-dependent selection for what type of microbes can live in an environment, and a one-time gavage of microbiota is unlikely to enable taxa that are part of a climax community to colonize. It's a bit too artificial of a measure of fitness with a bunch of troubling caveats.

6. The authors show that differences in density relate to differences in gene expression and host response. Unfortunately, there is no commentary on what the genes they identified were and whether there are plausible relationships between differences in microbiota and their density and host response. A fear is that such analyses are prone to false positives - even with correction for multiple comparisons. Given that the authors used host tissue from the cecum and the rest of the gastrointestinal track for these analyses, it would be nice to see confirmation that the DNA density in the feces correlated with its density in other locations.

7. Throughout the manuscript the authors measure density as the "ug of DNA per mg feces". It is unclear to me whether this was based on the wet or dry weight of the fecal material. I would argue that it should be on a dry basis in all of the analyses to control for differences in stool consistency. I know the authors have presented evidence that there's no correlation between density and water content, but it would be interesting to see the authors correct their data for moisture content. For example, is the variation in mouse microbiota density partly attributed to variation in water content? The colors in 1C do not allow me to easily discriminate between the 16 species, but it appears that there is more variation in moisture content than density for the mouse samples and several other species' samples as well.



Other comments...
1. L45 "rDNA" should be "rRNA gene" throughout the manuscript.

2. L65 "sixteen different mammals". Should be "sixteen mammal species"

3. Throughout the authors use the mean and SEM to report their results. These data do not appear to be normally distributed. I would find the results more compelling if they presented the median and interquartile range.



---

<br>

[^1]: I do not know the current status of this manuscript and have not been invited to review it for a journal. I have posted a copy of this review at [bioRxiv](https://www.biorxiv.org/content/early/2018/03/08/277095#comment-3820825610). Please post any comments there.
